gnovm/pkg/test/test.go

```diff
@@ -389,6 +414,133 @@ func (opts *TestOptions) runTestFiles(
 	return errs
 }
 
+func (opts *TestOptions) runFuzzFuncInTestFiles(
```

Comment:
duplicated with `runTestFiles` function. Wasn't there a way to handle this inside `runTestFiles`?